langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockLlamaChatModel.java

```diff
@@ -1,7 +1,7 @@
 package dev.langchain4j.model.bedrock;
 
 import dev.langchain4j.model.bedrock.internal.AbstractBedrockChatModel;
-import dev.langchain4j.model.bedrock.internal.BedrockChatModelResponse;
+
```

Comment:
nit: shouldn't it be together with other imports without empty line as before?

Reply:
@Wadimz done